packages/shared/forks/ReactFeatureFlags.native-oss.js

```diff
@@ -43,7 +43,7 @@ export const enableSuspenseCallback = false;
 export const disableLegacyContext = true;
 export const enableTrustedTypesIntegration = false;
 export const disableTextareaChildren = false;
-export const disableModulePatternComponents = false;
+export const disableModulePatternComponents = __NEXT_MAJOR__;
```

Comment:
Note that `__NEXT_MAJOR__` is just `false` for now in RN OSS defined above. This is just a reminder when we do that upgrade.